recipes/configs/llama2/13B_full.yaml

```diff
@@ -1,27 +1,19 @@
-# Config for multi-device full finetuning in full_finetune_distributed.py
-# using a Llama2 13B model
+# Config for multi-device with full_finetune_distributed.py using a Llama2 13B model
 #
-# This config assumes that you've run the following command before launching
-# this run:
-#   tune download --repo-id meta-llama/Llama-2-13b-hf \
-#   --hf-token <HF_TOKEN> \
-#   --output-dir /tmp/llama2-13b-hf
+# This config assumes that you've run the following command before launching:
+# $ tune download meta-llama/Llama-2-13b \
+#       --hf-token <HF_TOKEN> \
+#       --output-dir /tmp/llama2
 #
 # To launch on 4 devices, run the following command from root:
-#   tune run --nproc_per_node 4 full_finetune_distributed \
-#   --config llama2/13B_full \
+# $ tune run --nproc_per_node 4 full_finetune_distributed \
```

Comment:
Do we want to split out with an example single device command here too? (I guess for 13B this is maybe less likely anyways)

Reply:
Nvm I think these are not registered for the single-device recipes. Then maybe update the comment in 13B_lora.yaml?

Reply:
Update what comment? The topline comment says it's just for multi-device

Reply:
Sorry it's in the other yaml, commented at the specific lines